packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -26,21 +32,28 @@ import {
 } from "../../services/iModel";
 import { ViewerPerformance } from "../../services/telemetry";
 import type { ModelLoaderProps } from "../../types";
-import { IModelViewer } from "./";
 import { BackstageItemsProvider } from "../app-ui/providers";
-import { Presentation } from "@itwin/presentation-frontend";
+import { IModelViewer } from "./";
```

Comment:
importing like this from the barrel file is highly discouraged
import from the actual path itself